frontend/src/components/requests/actions.jsx

```diff
@@ -4,30 +4,18 @@ import './styles/actions.scss';
 
 export default class RequestsActions extends React.Component {
 
-    constructor(props) {
-        super(props);
-        this.handleChange = this.handleChange.bind(this);
-        this.doAction = this.doAction.bind(this);
-        this.state = { value: null };
-    }
-
-    handleChange(option) {
-        const value = this.props.events.find(e => e.name === option.value);
-        this.setState({ value });
-    }
-
-    doAction() {
-        if (!!this.state.value) {
-            this.props.onAction(this.state.value);
-        }
-    }
-
     render() {
 
-        const options = this.props.events
-            .map(event => ({
-                label: event.name,
-                action: () => this.props.onAction(event)
+        const statusOptions = this.props.statuses.items
+            .map(status => ({
+                label: status.name || 'No status',
```

Comment:
@danieljones740 can you capitalise the S just for consistency? e.g. **No Status**